### ISSUE TITLE
Ability to view raw metrics numbers as a dataframe view on the UI

### ISSUE THREAD

**Issue by arkkln** [aimhubio/aim#2172]
## 🚀 Feature

Currently the only way to explore metrics on UI is via Metrics Explorer, where metrics are displayed as line charts.
It would be great to add an ability to view the raw values of metrics as a dataframe view or table.

### Motivation

In order to learn deeper insights on the tracked metrics, it would be great to be able to explore **the raw numbers** of metrics. 
Additionally enabling common functionalities like pivoting and exporting the data, as well as enabling visualizations on the dataframe.

In other words - embedded google-sheets-like view. :) 

### Pitch

Add ability to switch from visualizing metrics as line charts to dataframe/table view.

### Alternatives

Query and export metrics data via SDK. 
Example: https://colab.research.google.com/drive/14rIAjpEyklf5fSMiRbyZs6iYG7IVibcI

<img width="654" alt="image" src="https://user-images.githubusercontent.com/13848158/190674954-9ef9dacd-4999-4986-8ad5-ea38639798a9.png">
